libvirt/cloudinit_def_test.go

```diff
@@ -161,6 +163,36 @@ ssh_authorized_keys:
 	}
 }
 
+func TestCreateCloudIsoViaPlugin(t *testing.T) {
+	var config = fmt.Sprintf(`
+
+   resource "libvirt_cloudinit" "test" {
+   name = "commoninit.iso"
+   local_hostname = "tango"
+   pool =           "default"
+   user_data =      "#cloud-config\nssh_authorized_keys: []\n" 
```

Comment:
I suggest you check if this resource has been created as well. Also, you could add tests with
* no explicit user_data (which will create an empty ssh_authorized_keys list)
* random user_data
* invalid user_data